crates/dpapi-native-transport/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "dpapi-native-transport"
 version = "0.1.0"
-edition = "2024"
+edition = "2021"
```

Comment:
note: I think it’s fine to stick to 2024. We’ll migrate to 2024 eventually, so we may as well reduce the future effort for it.